kernel/build.rs

```diff
@@ -6,11 +6,12 @@ where
     F: FnMut(&mut Command) -> &mut Command,
 {
     let mut command = Command::new(name);
-    println!("Running {:?}", command);
+    println!("Running {command:?}");
```

Comment:
See [clippy rule](https://rust-lang.github.io/rust-clippy/master/index.html#/uninlined_format_args). Readability change. Applies to all the changes like this.